buildtest/cli/report.py

```diff
@@ -479,12 +500,23 @@ def process_report(self):
 
     def print_format_fields(self):
         """Displays list of format field which implements command ``buildtest report --helpformat``"""
-        table = Table("[blue]Field", "[blue]Description", title="Format Fields")
+        table = Table(
+            "[blue]Field",
+            "[blue]Description",
+            title="Format Fields",
+        )
         for field, description in self.format_field_description.items():
             table.add_row(f"[red]{field}", f"[green]{description}")
 
         console.print(table)
 
+    def print_color_help(self):
+        """Displays list of colors supported by the rich console which implements command ``buildtest report --helpcolor``"""
+        table = Table("Color", title="Supported Colors")
+        for color_option in self.color_options:
+            table.add_row(f"[{color_option}]{color_option}")
+        console.print(table)
+
```

Comment:
Can you please remove this statement. This should be in a subsequent PR.

Reply:
removed